### PR TITLE
License update

### PR DESCRIPTION
**Pull request by jguz-pubnub**
refactor(license): update license information

### CODE REVIEW

File: PubNubSwift.podspec
Comment (seba-aln) [inline PubNubSwift.podspec:1]: just to be sure. You don't have to keep this structure? I mean `:file => 'LICENSE'` ??

Comment (jguz-pubnub) [reply PubNubSwift.podspec:1]: I removed it according to [this](https://guides.cocoapods.org/syntax/podspec.html#license) guideline that says:

> Unless the source contains a file named LICENSE.* or LICENCE.*, the path of the license file or the integral text of the notice commonly used for the license type must be specified. If a license file is specified, it either must be without a file extensions or be one of txt, md, or markdown.

Our source tree already contains the `LICENSE` file (without extension) in the root directory, so this should be valid.

Comment (seba-aln) [reply PubNubSwift.podspec:1]: Thanks for clarification

Comment (parfeon) [reply PubNubSwift.podspec:1]: @jguz-pubnub docs doesn't mention that root may contain file without extension to add automatically. But if you would like to specify it, the name shouldn't have an extension or has one of known extensions.
Or I got it in a wrong way?

Comment (jguz-pubnub) [reply PubNubSwift.podspec:1]: Hmm... now I started to have some doubts about that. I could try out the following and check whether `pod spec lint PubNubSwift.podspec` passes or not. Will this solve your worries?

``` s.license = { :type => 'PubNub Software Development Kit License', :file => 'LICENSE' }```

Comment (parfeon) [reply PubNubSwift.podspec:1]: @jguz-pubnub this is how I understood words from docs

Comment (jguz-pubnub) [reply PubNubSwift.podspec:1]: You're right - our source tree doesn't contain `LICENSE.*` or `LICENCE.*`. It has **LICENSE** file with no extension. So in that case, the path must be specified

**Review by parfeon — APPROVED** [review]
LGTM!

**Comment by jguz-pubnub** [comment]
@pubnub-release-bot release as 6.2.2

**Comment by jguz-pubnub** [comment]
@pubnub-release-bot release as 6.2.2

**Pull request merged**